css/style.css

```diff
@@ -455,3 +458,53 @@ form > textarea {
     border-color: transparent;
   }
 }
+
+@media screen and (max-width: 768px) {
+  #menu,
+  #menu:hover,
+  #menu:active {
+    justify-content: flex-end;
+    width: 18px;
+    border: none;
+    background-color: transparent;
+    background-image: url(../images/menu.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  #menu.menu-active {
+    background-image: url(../images/close.svg);
+  }
+
+  #navigation-links.nav-active {
+    position: absolute;
+    display: flex;
+    transition: transform 0.2s;
+    transform: translateX(0);
+    flex-direction: column;
+    align-content: center;
+    inset: 92px 0;
+    height: 90vh;
+    background-color: var(--color-menu);
+  }
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly adjust your styling so that your mobile menu have a width of `100%` and height of `100%` so that a user must click on the menu links before seeing the rest content in your page because right now, a user can ignore clicking the menu list items and just scroll down to view the contents of your page. 
Please, kindly fix this..

Reply:
**[REQUIRED CHANGES]**

- Also, Please add the white background and the black line to match the figma design. This could be achieved by placing your menu in a parent container so the parent has the white background and the black line. The menu which the child now takes up 90% height and 100% width of it parent container.

Reply:
> **[REQUIRED CHANGES]**
> 
> * Also, Please add the white background and the black line to match the figma design. This could be achieved by placing your menu in a parent container so the parent has the white background and the black line. The menu which the child now takes up 90% height and 100% width of it parent container.

Hello @zieeco, thanks for your review. I don't fully understand your second requirement, kindly provide a screenshot of the white background and black line you want me to edit, so I can make the changes.